hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -998,7 +999,7 @@ private void bindAWSClient(URI name, boolean dtEnabled) throws IOException {
    * @param parameters parameter object
    * @throws IOException on any IO problem
    */
-  private synchronized void createS3AsyncClient(S3ClientFactory clientFactory,
```

Comment:
we moved these out into their separate methods to fix spotbugs issues..cause they were not getting fixed by the addition to findbugs-exclude.xml. but if that's working now, we can revert these changes and intialise the s3Async client the same way as the sync one

Reply:
i don't know WTF is up with spotbugs. it works on the command line for me.